foundation-extensions/places/migration.md

```diff
@@ -83,12 +83,6 @@ static func getCurrentPointsOfInterest(_ closure: @escaping ([PointOfInterest])
 
 ### getLastKnownLocation
 
-{% tabs %}
-{% tab title="AEP 3.x \(Swift\)" %}
-{% hint style="info" %}
-If the SDK has no last known location, it will pass `nil` to the closure.
-{% endhint %}
-
 ```swift
```

Comment:
L86->L89 Tabs here seem not to pair up correctly, should this change be reverted?